test/http_fakes.dart

```diff
@@ -193,7 +195,7 @@ class FakeHttpResponse implements HttpResponse {
   }
 
   @override
-  void addError(error, [StackTrace stackTrace]) {
+  void addError(error, [StackTrace? stackTrace]) {
     // doesn't seem to be hit...hmm...
```

Comment:
😆

Reply:
Yeah, not much that I can do regarding this one. This is what the migration tool suggested, so I stuck with it.

Reply:
Oh I just thought the pre-existing comment was funny lol :D